app/components/agenda/draft-import.js

```diff
@@ -2,24 +2,24 @@ import Component from '@glimmer/component';
 import { inject as service } from "@ember/service";
 import { action } from "@ember/object";
 import { tracked } from "@glimmer/tracking";
-
-import { task } from "ember-concurrency-decorators";
+import { restartableTask } from "ember-concurrency-decorators";
 
 export default class AgendaDraftImportComponent extends Component {
   constructor(...args){
     super(...args);
-    this.getDrafts();
+    this.getDrafts.perform();
   }
   @service store;
   @tracked options;
   @tracked selected;
 
-  @action
-  async getDrafts(){
-    const containers=await this.store.query('document-container', {
+  @restartableTask
+  * getDrafts(){
+    const containers=yield this.store.query('document-container', {
       include: 'current-version,ontwerp-besluit-status',
       'filter[status][:id:]': 'c02542af-e6be-4cc6-be60-4530477109fc',
-      'filter[folder][:id:]': 'ae5feaed-7b70-4533-9417-10fbbc480a4c'
+      'filter[folder][:id:]': 'ae5feaed-7b70-4533-9417-10fbbc480a4c',
+      'page[size]': '1000'
```

Comment:
I'd rather have a proper search on the component instead of just requesting 1000 documents.

Reply:
so not 100% sure what you mean. Do you want me to do a filter query onchange of the search bar in powerselect?